static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -58,6 +113,18 @@ const Context = ({
   registersMeta,
 }: Props) => {
   const isMobile = isMobileLanguage(event);
+  const {projects} = useProjects();
+  const project = useMemo(
+    () => projects.find(p => p.id === event.projectID),
+    [projects, event]
+  );
+
+  const {data, isLoading} = useStacktraceLink({
+    event,
+    frame,
+    orgSlug: organization?.slug || '',
+    projectSlug: project?.slug,
+  });
```

Comment:
we should disable this when the feature flag is not present 
might need to add a second argument to useStacktraceLink like this one https://github.com/getsentry/sentry/blob/7b871842ba271afb624570b52187009841483837/static/app/utils/useCommitters.tsx#L23

useQuery options takes a `enabled` property

Reply:
this needs to be disabled when expanded is false or it will make a query for every frame